src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -378,12 +390,14 @@ def _fill_ftp_publishing_url(cmd, webapp, resource_group_name, name, slot=None):
     return webapp
 
 
-def _format_linux_fx_version(custom_image_name):
+def _format_linux_fx_version(custom_image_name, custom_prefix=None):
```

Comment:
Unless `custom` is a agreed-on term, shall we rename to "container_config_type" to make it more explicit?